### ISSUE TITLE
Show default values in Config tab

### ISSUE THREAD

**Issue by Croway** [KaotoIO/kaoto-ui#1307]
### Please describe the feature that you want to propose

Would it be possible to show the default value like https://camel.apache.org/camel-kamelets/3.20.x/timer-source.html (period property) in Kaoto component/kamelet config tab?

![image](https://user-images.githubusercontent.com/34543311/220655584-33c4131c-aab5-4fe1-80b2-2e66535f9d0b.png)

I'm using http://kaoto-kaoto.apps.smaug.na.operate-first.cloud/

**Comment by kahboom** [comment]
Hey @Croway, thanks for opening this issue. Just want to check without assuming--is this the same as issue https://github.com/KaotoIO/kaoto-ui/issues/1154 or did you mean something different?

**Comment by Croway** [comment]
@kahboom yes, the issue seems similar, but if I understand [#1154 ](https://github.com/KaotoIO/kaoto-ui/issues/1154) correctly, the default value should be set into the input field. But if you don't want to have the input field (like Period in this example) pre filled, I think that adding this information into the help ? may help.

If you are going with [#1154 ](https://github.com/KaotoIO/kaoto-ui/issues/1154), then this issue can be closed. I'm sorry, my search skills on github issues are awful.

**Comment by kahboom** [comment]
@Croway Ah okay I see what you mean. That's a good idea, because if you override the default value then you should still have a reference to what the default was before you changed it. Usually this kind of thing goes in the placeholder of the field when you erase the value (though we don't currently do this), but it could be nice to have it visible somewhere like where you mentioned, without having to delete the values.

I will keep this issue open because I think it's a different thing.

**Comment by lhein** [comment]
I would (maybe additionally to the help hover) add the default value as a placeholder value for a text field so it is displayed if there is no other value specified but disappears once you click into the field to enter something.

**Comment by lhein** [comment]
for checkboxes we should check if the default says true

**Comment by lhein** [comment]
tbh I have no strong opinion on whether we use placeholders or directly put the default values into the fields. In the end defaults will be filtered out anyways by the backend.
